### DISCUSSION TITLE
%E5%89%8D%E8%A8%80/

### DISCUSSION THREAD

**Discussion by giscus** [SurviveXJTU/SurviveXJTU.github.io#1] [Announcements]
# %E5%89%8D%E8%A8%80/

西安交大生存指南，更适合西交大学子的生存指南。

https://survivexjtu.github.io/%E5%89%8D%E8%A8%80/

**Comment by Axi404** [comment]
欢迎大家在本页打卡！欢迎来到西安交大！

**Comment by YRmaxwell** [comment]
打卡！希望越办越好，做大做强！

**Reply by Axi404** [reply]
感谢你的支持！

**Comment by weaver618** [comment]
三楼打卡！作者太强了！必须支持！

**Reply by Axi404** [reply]
感谢你的支持，欢迎持续关注！

**Comment by ChiyoYuki** [comment]
感谢作者贡献，友校学生前来参考学习，在尝试采用类似方案部署本校手册中

**Reply by Axi404** [reply]
感谢支持！
